### PR TITLE
redoing: namespace get_template_part action hook

### PR DESCRIPTION
**Pull request by cliffordp**
by adding a new action hook instead of replacing the existing one

replacement for the pull request that _was_ at https://github.com/GaryJones/Gamajo-Template-Loader/pull/25

### CODE REVIEW

File: class-gamajo-template-loader.php [hunk 1]
```diff
@@ -110,6 +110,7 @@ public function __destruct() {
 		public function get_template_part( $slug, $name = null, $load = true ) {
 			// Execute code for this part.
 			do_action( 'get_template_part_' . $slug, $slug, $name );
+			do_action( 'get_template_part_' . $this->filter_prefix . '_' . $slug, $slug, $name );
```
Comment (GaryJones) [inline class-gamajo-template-loader.php]: I think it should be:
```php
do_action( $this->filter_prefix . '_get_template_part_' . $slug, $slug, $name );
```
as that's how the `filter_prefix` property is used in two existing filters.

**Review by GaryJones — CHANGES_REQUESTED** [review]

**Review by GaryJones — APPROVED** [review]

**Comment by GaryJones** [comment]
Thank you :-)

**Comment by cliffordp** [comment]
Thank _you_!

Also wanted to remind you of the version bump in both the file's comments and the tag on github.

**Comment by GaryJones** [comment]
@cliffordp Thanks for the nudge :-)

https://github.com/GaryJones/Gamajo-Template-Loader/releases/tag/1.3.0

**Pull request merged**